src/main/java/com/exceptionless/exceptionlessclient/queue/DefaultEventQueue.java

```diff
@@ -137,11 +136,13 @@ public void process() {
       log.info(
           String.format("Sending %s events to %s", events.size(), configuration.getServerUrl()));
       SubmissionResponse response = submissionClient.postEvents(events);
+      if (response.hasException()) {
+        log.error("Error submitting events from queue", response.getException());
+        suspendProcessing();
+        return;
+      }
       processSubmissionResponse(response, storedEvents);
       eventPosted(response, events);
-    } catch (SubmissionClientException e) {
```

Comment:
Might be worth keeping this just in case something throws. But if we are confident there are no issues it can be removed.

Reply:
I am now sending the exception encapsulated inside the response object.